README.md

```diff
@@ -134,6 +134,10 @@ only valid for 'base'.
 - if set to 'file' it will create a child process for each mocha file (alternative to `-F` CLI arg)
 - if set to 'data' it will create a child process for each object key under `base.data` (alternative to the `-D` CLI arg)
 
+### `base.reports`
+
+Recommended to set this as `path:report`, which will create a `report` directory beneath your base directory. See `Reporters` below.
```

Comment:
The `Reporters` section was renamed to `Reporting`, so it might be better for the last sentence to read
```
See `Reporting` section below.
```